lib/commands/intent.js

```diff
@@ -176,54 +171,77 @@ commands.mobileStartActivity = async function mobileStartActivity (opts = {}) {
   if (!_.isNil(display)) {
     cmd.push('--display', display);
   }
-  cmd.push(...(parseCommonIntentArguments(opts)));
-  cmd.push(intent);
+  cmd.push(...(parseIntentSpec(opts)));
+  return await this.adb.shell(cmd);
+};
+
+/**
+ * @typedef {Object} BroadcastOptions
+ * @property {?string|number} user ['all'] - The user ID for which the broadcast is sent.
+ * The `current` alias assumes the current user ID.
+ * @property {?string} receiverPermission - Require receiver to hold the given permission.
+ * @property {?string} intent - The name of the intent to broadcast to, for example
+ * `com.some.package.name/.YourServiceSubClassName`.
+ * @property {?string} action - Action name
+ * @property {?string} uri - Unified resource identifier
+ * @property {?string} mimeType - Mime type
+ * @property {?string} identifier - Optional identifier
+ * @property {?string|Array<string>} categories - One or more category names
+ * @property {?string} component - Component name
+ * @property {Array<string|Array<string>>} extras - Optional intent arguments.
+ * See above for the detailed description.
+ * @property {?string} flags - Intent startup-specific flags as a hexadecimal string.
+ * See above for the detailed description.
+ */
+
+
+/**
+ * Send a broadcast intent.
+ *
+ * @param {BroadcastOptions} opts
+ * @returns {string} The command output
+ * @throws {Error} If there was a failure while starting the activity
+ * or required options are missing
+ */
+commands.mobileBroadcast = async function mobileBroadcast (opts = {}) {
```

Comment:
ah, good idea.
I remember I called this broadcast to automate push notification (via broadcast receiver in android app)